src/action.js

```diff
@@ -54,60 +69,64 @@ function build(programInfo) {
   /**
    * TODO: Generate other HTML files from directory just like index.abell
    */
-  
-  // GENERATE OTHER HTML FILES
-  generateHTMLFile('index', programInfo);
-  if (programInfo.logs == 'complete') console.log(`...Built index\n`);
 
-  if (programInfo.logs == 'minimum') console.log(`${boldGreen('>>>')} Files built.. ✨`);
-}
+  // GENERATE OTHER HTML FILES FROM ABELL
+  abellFiles.map((file) => {
```

Comment:
I guess you meant to do `.forEach` here. .map is supposed to be use when you want to return something. 

Also, if it is a plain array then I will suggest `for (const file of abellFiles) {}` instead

Reply:
Fixed this ✅

Reply:
Can you check this once again? I guess this isn't resolved